samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,394 @@
+:orphan:
+
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to update and boot the nRF54H20 System-on-Chip (SoC) using the Software Update for Internet of Things (SUIT) procedure on the application and radio cores of the SoC.
```

Comment:
Link `Software Update for Internet of Things (SUIT)` to a generic SUIT intro page (in the NCS docs or external) or to a term in the glossary (which we also need anyway, actually).